app/models/enumeration.rb

```diff
@@ -48,26 +48,12 @@ class Enumeration < ApplicationRecord
   scope :shared, -> { where(project_id: nil) }
   scope :active, -> { where(active: true) }
 
-  # let all child classes have Enumeration as its model name
-  # used to not having to create another route for every subclass of Enumeration
-  def self.inherited(child)
-    # in the long run we probably want to have seperate pages, time entry activities are first so we skipt it here
-    enumerations_with_own_settings = %w(TimeEntryActivity)
-
-    unless child.name.in?(enumerations_with_own_settings)
-      child.instance_eval do
-        def model_name
-          Enumeration.model_name
-        end
-      end
-    end
-    super
-  end
-
```

Comment:
We can also get rid of the loading of all subclasses for development mode that is happening in lines 178-181. We are no longer iterating over all subclasses